development-tools/clinic.rst

```diff
@@ -1495,6 +1495,27 @@ You can still use a self converter, a return converter, and specify
 a *type* argument to the object converter for :c:macro:`METH_O`.
 
 
+How to convert var-positional parameter functions
+-------------------------------------------------
+
+To convert a var-positional parameter function,
+prepending the parameter name with ``*`` ,
+and the parameter should use the ``object`` converter::
+
+   /*[clinic input]
+   var_positional_sample
+
+       foo: int
+       *args: object
+   [clinic start generated code]*/
+
+The implementation function will receive var-positional arguments
+as a tuple, you can either use the tuple directly
+or parsing them into C types using :c:func:`!PyArg_Parse*` functions.
```

Comment:
```suggestion
The implementation function will receive var-positional arguments
as a tuple; you can either use the tuple directly or parse it into
C types via :c:func:`PyArg_ParseTuple` or :c:func:`PyArg_UnpackTuple`.
```

Am I right concerning the functions to use? I don't think we have other
functions that would be relevant in this specific case.

Reply:
I think it's OK to skip this part. It's important to mention that it's a tuple; for figuring out what to do with the tuple we can point to the C API documentation.
```suggestion
The implementation function will receive var-positional arguments
as a tuple.
```

Reply:
I'm also ok if we omit the C API mention!